### PR TITLE
python-exposing-properties

### PR DESCRIPTION
**Pull request by Kaszana102**
because previous rebase failed

### CODE REVIEW

File: pkg/game/game.go [hunk 1]
```diff
@@ -108,12 +110,26 @@ func (game *Game) DeepCloneWithLog(log logger.Logger) (*Game, error) {
...
 
 func (game *Game) Serialized() SerializedGame {
+
+	// serialize serializedPlayers
+	serializedPlayers := []elements.SerializedPlayer{}
+	for _, player := range game.players {
+		serializedPlayers = append(serializedPlayers, player.Serialized())
+	}
+
+	// create serialzied tiles
```
Comment (bstrzelecki) [inline pkg/game/game.go:120]: typo

File: python_bindings/carcassonne_engine/player.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from ._bindings import elements as _go_elements  # type: ignore[attr-defined] # no stubs
+
+__all__ = "Player"
```
Comment (Jackenmen) [inline python_bindings/carcassonne_engine/player.py:3]:
```suggestion
__all__ = ("Player",)
```

File: python_bindings/carcassonne_engine/requests.py [hunk 3]
```diff
@@ -13,6 +13,8 @@
     "MoveWithState",
 )
 
+from .placed_tile import PlacedTile
```
Comment (Jackenmen) [inline python_bindings/carcassonne_engine/requests.py:16]: Put this alongside the rest of the imports (i.e. above `__all__`).

File: pkg/game/game.go [hunk 4]
```diff
@@ -21,10 +22,11 @@ type SerializedGame struct {
 	CurrentTile         tiles.Tile
 	ValidTilePlacements []elements.PlacedTile
 	CurrentPlayerID     elements.ID
-	Players             []elements.Player
+	Players             []elements.SerializedPlayer
 	PlayerCount         int
 	Tiles               []elements.PlacedTile
 	TileSet             tilesets.TileSet
+	BinaryTiles         []binarytiles.BinaryTile // contains info about all placed tiles, not placed tiles are equal to 0
```
Comment (Jackenmen) [inline pkg/game/game.go:29]: I wonder, isn't this a bit excessive? The API user can already call `to_bits()` on elements of the `Tiles` slice.

**Review by Jackenmen — DISMISSED** [review]
I only did a preliminary look so far but I noticed your code is currently exposing internal types such as `Slice_elements_PlacedTile` in public APIs. We want to avoid that - please convert them to the equivalent Python types (e.g. `list[PlacedTile]`) instead.

File: python_bindings/carcassonne_engine/requests.py (hunk 3 above)
Comment (Kaszana102) [reply python_bindings/carcassonne_engine/requests.py]: moved

File: python_bindings/carcassonne_engine/player.py (hunk 2 above)
Comment (Kaszana102) [reply python_bindings/carcassonne_engine/player.py]: changed

File: pkg/game/game.go (hunk 1 above)
Comment (Kaszana102) [reply pkg/game/game.go]: changed

**Review by bstrzelecki — APPROVED** [review]

**Review by Lenetis — APPROVED** [review]

**Pull request merged**